static/src/javascripts/projects/common/modules/commercial/templates/acquisitions-epic-control.js

```diff
@@ -1,8 +1,20 @@
 // @flow
 import { appendToLastElement } from 'lib/array-utils';
 
+const buildFooter = (footer: string[]): string =>
```

Comment:
Rather than an array of strings, how about creating a `type` called something like EpicFooterLine which looks something like:

```
declare type EpicFooterLine = {
   startOfSentence: string, 
   blueEndOfSentence: string
}
```

That way you don't have to assume that only the first word would be blue. Just an idea, as I can foresee us potentially wanting to highlight the first `x` words (for example, in your screenshot I imagine that `The Guardian` would be all blue, rather than just `The`

Reply:
Good idea. I'm not sure how this could be configured from the google sheet though

Reply:
Yeah, that would definitely be a nicer model. But you either need HTML or some sort of custom markdown-esque syntax to do this simply in the google doc

Reply:
The requirements have changed and this logic is no longer required!
Instead we're just doing alternating colours, see latest commit